### ISSUE TITLE
Create a BUILD file for accessibility/

### ISSUE THREAD

**Issue by fsharpasharp** [oppia/oppia-android#2643]
This task is part of the build system migration from Gradle to Bazel. A prerequesite before working on this issue is that you are able to build Oppia using Bazel. Check out [Oppia Bazel guide](https://github.com/oppia/oppia-android/wiki/Oppia-Bazel-Guide) for Bazel terminology and more of what it does.

Create a BUILD.bazel file in 

`src/main/java/org/oppia/android/util/accessibility/`

This package does not depend on any other packages inside of Oppia itself. Create a library for each file in the package, if possible, and add the dependency artifacts that are required based on the imports in the file.

## Create the libraries

I'm using the filename interchangebly with the library here.

`AndroidAccessibilityManager.kt` should be a private library since it is only used by `AccessibilityModule.kt`.

`CustomAccessibilityManager.kt` is referenced by `AccessibilityModule.kt`, `FakeAccessibilityModule.kt` and externally. It should be a public library and be a deps dependency of the two first libraries.

## Confirm that the package builds

When you've added the libraries, confirm that you can build the package by using Bazel to build the libraries. If you're in the  directory in your terminal, run

`bazel build :all` 

to build all your libraries.

## Replace srcs dependencies in the module BUILD files

Add the required deps dependencies to the module's BUILD.bazel file.

Since `FakeAccessibilityModule.kt` was also used in some tests, we will have to find the corresponding libraries there as well to replace the srcs dependency with the new deps dependency.

`CustomAccessibilityManager.kt` is used in the app module. Look for the libraries that have `//utility` in its deps dependencies and check if it needs the new library you created as a dep dependency.

## Confirm that the app builds

Now you can confirm that the package works by building the app successfuly.

**Comment by FareesHussain** [comment]
@fsharpasharp How do I add an artifact that is related to Android SDK 
For Example `AndroidAccessibilityMangaer.kt` uses a import
```
import android.view.accessibility.AccessibilityManager
```
Which is related to Android SDK

**Comment by fsharpasharp** [comment]
@FareesHussain I think that import would not require an explicit dependency since it's part of the Android SDK, as you say.

**Comment by BenHenning** [comment]
Taking this per #2775.
